openmetadata-ui/src/main/resources/ui/src/utils/FilterUtils.js

```diff
@@ -49,10 +49,7 @@ export const getFilterString = (
     const filter = filters[key];
     filter.forEach((value) => {
       const modifiedKey = prepareModifiedKey(key, restrictKeyModification);
-      const modifiedValue = key === 'tags' ? `"${value}"` : value;
-      modifiedFilter.push(
-        `${modifiedKey.split(' ').join('_')}:${modifiedValue}`
-      );
+      modifiedFilter.push(`${modifiedKey.split(' ').join('_')}:${value}`);
```

Comment:
Why did we remove this?
```js
const modifiedValue = key === 'tags' ? `"${value}"` : value;
```

Reply:
As we are handling quoting for all the queries in other function I removed this as it was adding double quotes.